docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/EventSubscriber/RedirectSubscriber.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Drupal\cgov_core\EventSubscriber;
+
+use Drupal\Core\Path\CurrentPathStack;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+
+/**
+ * Event subscriber subscribing to KernelEvents::REQUEST.
+ */
+class RedirectSubscriber implements EventSubscriberInterface {
+
+  /**
+   * Current path getter because paths > routes for users.
```

Comment:
Why is the path better than a route?
Wouldn't it be easier to use the route and check if the name matches user.reset?

Reply:
I initially tried to get the current route, but it gives null. That's why i used path.

Reply:
So there was another module I found that had a good example of checking the route. However, this [stack overflow post](https://drupal.stackexchange.com/questions/272182/disable-user-login-register-and-password-reset) looks much simpler to maintain. 

(Which I should have thought to suggest as we have other code where we modify _permissions to a route... :( )

Reply:
This (https://drupal.stackexchange.com/questions/272182/disable-user-login-register-and-password-reset) looks nice.
So if we use the above one, we don't need the redirect right? Please advise.